diff -urNp a/etc/gimprc.in b/etc/gimprc.in
--- a/etc/gimprc.in	2019-08-20 13:03:43.195089141 +0200
+++ b/etc/gimprc.in	2019-08-20 13:04:09.109353485 +0200
@@ -158,7 +158,7 @@
 # Where to look for fonts in addition to the system-wide installed fonts. 
 # This is a colon-separated list of folders to search.
 # 
-# (font-path "${gimp_dir}/fonts:${gimp_data_dir}/fonts")
+(font-path "${gimp_dir}/fonts")
 
 # Specify a default brush.  The brush is searched for in the specified brush
 # path.  This is a string value.
