diff -up gimp-2.10.14/etc/gimprc.in.cm-system-monitor-profile-by-default gimp-2.10.14/etc/gimprc.in
--- gimp-2.10.14/etc/gimprc.in.cm-system-monitor-profile-by-default	2019-10-26 21:46:48.000000000 +0200
+++ gimp-2.10.14/etc/gimprc.in	2019-11-04 13:17:23.322750636 +0100
@@ -316,9 +316,9 @@
 
 # Defines the color management behavior.  This is a parameter list.
 # 
-# (color-management
+(color-management
 #     (mode display)
-#     (display-profile-from-gdk no)
+      (display-profile-from-gdk no)
 #     (display-rendering-intent relative-colorimetric)
 #     (display-use-black-point-compensation yes)
 #     (display-optimize yes)
