--- b/app/sanity.c
+++ a/app/sanity.c
@@ -502,7 +502,7 @@
 
 #define GEGL_REQUIRED_MAJOR 0
 #define GEGL_REQUIRED_MINOR 4
+#define GEGL_REQUIRED_MICRO 34
-#define GEGL_REQUIRED_MICRO 36
 
   gegl_get_version (&gegl_major_version,
                     &gegl_minor_version,
--- b/configure.ac
+++ a/configure.ac
@@ -60,7 +60,7 @@
 m4_define([freetype2_required_version], [2.1.7])
 m4_define([gdk_pixbuf_required_version], [2.30.8])
 m4_define([gegl_major_minor_version], [0.4])
+m4_define([gegl_micro_version], [34])
-m4_define([gegl_micro_version], [36])
 m4_define([gegl_required_version],
           [gegl_major_minor_version.gegl_micro_version])
 m4_define([gexiv2_required_version], [0.12.2])
--- b/meson.build
+++ a/meson.build
@@ -332,7 +332,7 @@
 freetype2         = dependency('freetype2',          version: '>='+freetype2_minver)
 gdk_pixbuf_minver = '2.30.8'
 gdk_pixbuf        = dependency('gdk-pixbuf-2.0',     version: '>='+gdk_pixbuf_minver)
+gegl_minver       = '0.4.34'
-gegl_minver       = '0.4.36'
 gegl              = dependency('gegl-0.4',           version: '>='+gegl_minver)
 gexiv2_minver     = '0.12.2'
 gexiv2            = dependency('gexiv2',             version: '>='+gexiv2_minver)
